server/data/quotes.json

```diff
@@ -275,6 +275,10 @@
     "author": "Jhumpa Lahiri",
     "sentence": "A foreign language is like a frail, delicate muscle. If you don't use it, it weakens."
   },
+  {
+    "author": "Johann Wolfgang von Goethe",
+    "sentence": "Those who know nothing of foreign languages know nothing of their own."
```

Comment:
This is a duplicate quote (see line 295), please add a different quote.

Reply:
Thanks for pointing, I was using ctrl+f against the whole sentence, didn't know that the same quote may exist rephrased.

Reply:
Thanks for changing! :)